.github/workflows/release-docker.yml

```diff
@@ -41,9 +41,9 @@ jobs:
         id: version
         run: |
           IFS="." read -r MAJOR MINOR PATCH <<< ${{ github.event.inputs.version_number }}
-          echo "::set-output name=major::$MAJOR"
-          echo "::set-output name=minor::$MINOR"
-          echo "::set-output name=patch::$PATCH"
+          echo "major=$MAJOR >> $GITHUB_OUTPUT"
+          echo "minor=$MINOR >> $GITHUB_OUTPUT"
+          echo "patch=$PATCH >> $GITHUB_OUTPUT"
```

Comment:
Move the quotes to before you pipe to $GITHUB_OUTPUT

Reply:
Fixed. Thanks!